test/e2e/gitops_service_test.go

```diff
@@ -1,12 +1,13 @@
 package e2e
 
 import (
-	"fmt"
+	"strings"
 	"testing"
 	"time"
 
 	goctx "context"
```

Comment:
I'm curious about why this is being done?

Shortening from context to goctx? Conflict with something more important?

Reply:
Maybe to avoid confusion. Context by operator-sdk `framework.context` is being used in the same file. But there's no conflict. We can change it